00-locals.tf

```diff
@@ -25,16 +25,23 @@ locals {
 
   common_config_patch = {
     cluster = {
+    
       id          = var.cluster_id,
       clusterName = var.cluster_name,
       apiServer = {
-        certSANs = [
-          module.elb_k8s_elb.elb_dns_name
-        ]
+        extraArgs = {
+          cloud-provider = "external"
```

Comment:
```suggestion
```

kube-apiserver doesn't have a `cloud-provider` extraArg, according to https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/.